pip/pip-370.md

```diff
@@ -0,0 +1,48 @@
+# PIP-370: An optional flag: disable the replicators to automatically trigger topic creation
+
+# Background knowledge
+
+Users using Geo-Replication backup data across multi clusters, and Admin APIs related to Geo-Replication and internal replicators of brokers will trigger topics auto-creation between clusters.
+- For partitioned topics.
+  - After enabled namespace-level Geo-Replication: broker will create topics on the remote cluster automatically when calling `pulsar-admin topics create-partitioned-topic`. It does not depend on enabling `allowAutoTopicCreation`.
+  - When enabling topic-level Geo-Replication on a partitioned topic: broker will create topics on the remote cluster automatically. It does not depend on enabling `allowAutoTopicCreation`.
+- For non-partitioned topics and partitions of partitioned topics.
+  - The internal Geo-Replicator will trigger topics auto-creation for remote clusters. **(Highlight)** It depends on enabling `allowAutoTopicCreation`. In fact, this behavior is not related to Geo-Replication, it is the behavior of the internal producer of Geo Replicator,   
+
+# Motivation
+
+But if users want to maintain pulsar resources manually, there is no choice that prevent pulsar resources creation affects each other between clusters.
+
+**Passed solution**: disable `allowAutoTopicCreation`, the APIs `pulsar-admin topics create-partitioned-topic` still create topics on the remote cluster when enabled namespace level replication, the API `enable topic-level replication` still create topics, And the internal replicator will keep printing error logs due to a not found error.
```

Comment:
A bit of contemplation: we should maintain consistency in the behavior of creating partitioned topics and non-partitioned topics in remote clusters. Both should use the admin API.

Reply:
Yes, see the details: https://github.com/apache/pulsar/pull/23124#discussion_r1704293037